ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -1751,4 +1759,35 @@ Err:
         End If
     End Sub
 
+    Private Sub CombEW_SelectedIndexChanged(sender As Object, e As EventArgs) Handles CombEW.SelectedIndexChanged
+        Dim myLon As Double
+        If CombEW.SelectedItem = "W" Then
+
+            myLon = (-1) * (txtDegreesLon.Text + txtMinutesLon.Text / 60 + txtSecondsLon.Text / 3600)
```

Comment:
@isedwards -I am not sure if you get the point I meant clearly;
![image](https://user-images.githubusercontent.com/10340170/27901226-ec047eca-6239-11e7-9da6-b686f4f7e6a6.png)
I don't want 0s to appear by default in the form above but rather empty text boxes.